ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml

```diff
@@ -300,7 +303,7 @@ tests:
 - as: e2e-gcp-libvirt-cert-rotation
   cron: '@daily'
   steps:
-    cluster_profile: gcp
+    cluster_profile: gcp-openshift-gce-devel-ci-2
```

Comment:
does this one need a base domain change too?

Reply:
No, GCP is done in a smarter way and does not need this.